brickflow/engine/compute.py

```diff
@@ -106,6 +106,19 @@ def validate(self) -> None:
             (self.min_workers is not None and self.max_workers is not None)
             and (self.min_workers > self.max_workers)
         ), "Min workers should be less than max workers"
+        assert not (
```

Comment:
It would be nice to capture the error when the workflow is being built rather than during the terraform apply. Faster feedback to the user and cleaner error than terraform jargon. What do you think @asingamaneni ?

Reply:
Yeah I think it would be better to do it earlier. We can create a separate issue to refactor these.

Reply:
@BrendBraeckmans, this makes sense. We would need an issue to clean up and create proper exceptions.

Reply:
Will make some time later today to create the high level issue